index.js

```diff
@@ -4,22 +4,47 @@
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function addNewStore(stores, store) {}
+function addNewStore(stores, store) {
+  stores[stores.length]= store;
```

Comment:
It's best practice to use .push instead of assignment for adding a new element to the end of an array